api/app/Models/Pool.php

```diff
@@ -27,6 +27,7 @@
  * @property string $advertisement_language
  * @property boolean $is_published
  * @property string $stream
+ * @property string #process_number
```

Comment:
does the "#" not break anything?

Reply:
Oops. No, this is just documentation. It's not required at all.